.env

```diff
@@ -0,0 +1,4 @@
+MONGO_URI = 'mongodb+srv://social_network:social_network@cluster0.clqy6jt.mongodb.net/nestjs-test?retryWrites=true&w=majority'
```

Comment:
We never commit .env file, instead we make .env.sample file to only store the names of the variables.